packages/core/src/plugins/default/markbind-plugin-plantuml.js

```diff
@@ -84,12 +86,23 @@ module.exports = {
 
   beforeSiteGenerate: () => {
     processedDiagrams.clear();
+    graphvizCheckCompleted = false;
   },
 
   processNode: (pluginContext, node, config) => {
     if (node.name !== 'puml') {
       return;
     }
+    if (config.plantumlCheck && !graphvizCheckCompleted) {
+      exec('dot -V', (error, stdout, stderr) => {
```

Comment:
```suggestion
      exec(`java -jar "${JAR_PATH}" -testdot`, (error, stdout, stderr) => {
```

Would this be better since it is the recommended testing method on the [plantuml site](https://plantuml.com/graphviz-dot#fa6a5a3224d7da66)? The original should work as well for most cases (except when Graphviz is installed in a custom path).